src/app/features/test-records/components/base-test-record/base-test-record.component.html

```diff
@@ -25,6 +25,18 @@
         ></app-defects>
       </app-accordion>
 
+      <app-accordion *ngSwitchCase="'requiredStandards'" [id]="template.name" [title]="template.label">
+        <app-required-standards
+          class="section"
+          [id]="template.name"
+          [isEditing]="isEditing"
+          [requiredStandards]="getDefects$(testResult.vehicleType) | async"
```

Comment:
This will be added in a future ticket, just left it as defects until then, no harm caused.